tests/integration/url_generator_only/test_url_generator_only.py

```diff
@@ -50,3 +50,62 @@ def test_expected_integration_sample():
             '''
             assert queries[0].query_string != ""
 
+    
+def expected_test_for_get():
+    path_to_metadata = Path(__file__).parent.joinpath("metadata-northwind-v2.xml")
+    metadata_file_contents = path_to_metadata.read_bytes()
+    restrictions = RestrictionsGroup(None)
+    get_builder = DirectBuilder(metadata_file_contents, restrictions,"GET")
+    get_entities = get_builder.build()
+    queryable_factory = SingleQueryable
+    queries_list = []
+    queries_list.clear()
+    for queryable in get_entities.all():
+        entityset_urls_count = len(queryable.entity_set.entity_type.proprties())
+        for _ in range(entityset_urls_count):
+            q = queryable_factory(queryable, logger, 1)
+            queries = q.generate()
+            queries_list.append(queries[0].query_string)
+    queries_list=set(queries_list)
+    choice = queries_list.pop()
+    if("filter" in choice or "expand" in choice or "startswith" in choice or "replace" in choice or "substring" in choice or "inlinecount" in choice):
+        return True
+    else:
+        return False
+    
+
+def expected_test_for_delete():
```

Comment:
Functions that are executed as unit tests by pytest must start with string "test".  When you executed your code locally, you should be surprised that the number of passing tests is still just 1.

Documentation is here:  https://docs.pytest.org/en/stable/

example names 
"test_direct_builder_http_get
"test_direct_builder_http_delete"

Reply:
ok